docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -84,3 +84,77 @@ type: Opaque
 ----
 
 Failure to do so can cause data loss.
+
+[id="{p}-{page_id}-pod-updates"]
+== Pods are not replaced after a configuration update
+
+The update of an existing Elasticsearch cluster configuration can fail because the operator is unable to apply the changes required while replacing the pods of a given Elasticsearch cluster.
+A key indicator is when the Phase of the Elasticsearch resource is in `ApplyingChanges` state for too long:
+
+[source,sh]
+----
+kubectl get es
+
+NAME                  HEALTH   NODES    VERSION   PHASE            AGE          
+elasticsearch-sample  yellow   2        7.9.2     ApplyingChanges  36m
+----
+
+Possible causes include:
+
+* The Elasticsearch cluster is not healthy
++
+[source,sh]
+----
+kubectl get elasticsearch
+
+NAME                                                              HEALTH   NODES   VERSION   PHASE   AGE
+elasticsearch.elasticsearch.k8s.elastic.co/elasticsearch-sample   yellow   1       7.9.2     Ready   3m50s
+----
++
+In this case, you have to link:{ref}/cluster-allocation-explain.html[check] and fix your shard allocations.  The link:{ref}/cluster-health.html[cluster health], link:{ref}/cat-shards.html[cat shards], and link:https://www.elastic.co/guide/en/cloud-on-k8s/1.3/k8s-deploy-elasticsearch.html#k8s_monitor_cluster_health_and_creation_progress[get Elasticsearch] APIs can assist in tracking the shard recover process.
```

Comment:
Updated the link to use an anchor within this doc so it continues to follow the same version of the docs as there are more releases.
```suggestion
In this case, you have to link:{ref}/cluster-allocation-explain.html[check] and fix your shard allocations.  The link:{ref}/cluster-health.html[cluster health], link:{ref}/cat-shards.html[cat shards], and <<{p}-elasticsearch-monitor-cluster-health,get Elasticsearch>> APIs can assist in tracking the shard recover process.
```